tests/unit/libs/046-sanitize.js

```diff
@@ -460,4 +460,43 @@ describe('sanitize', () => {
       })
     })
   })
+
+  describe('bbox', () => {
+    it('should reject an incomplete bbox', async () => {
+      const req = { query: { bbox: '[0, 0, 1]' } }
+      const configs = { bbox: {} }
+      await sanitize(req, {}, configs)
+      .then(shouldNotBeCalled)
+      .catch(err => {
+        err.message.should.startWith('invalid bbox')
+      })
+    })
+
+    it('should reject bbox with a higher minLng than maxLng', async () => {
+      const req = { query: { bbox: '[0, 2, 1, 1]' } }
+      const configs = { bbox: {} }
+      await sanitize(req, {}, configs)
+      .then(shouldNotBeCalled)
+      .catch(err => {
+        err.message.should.startWith('invalid bbox')
+      })
+    })
+
+    it('should reject bbox with a higher minLat than maxLat', async () => {
```

Comment:
so no limit. One can query the whole world theoretically?

Reply:
there is a hard coded limit of 500 in `server/lib/search_by_position.js`, so searching the whole world is possible, but only the first 500 results will be returned. We could pass it a `limit` parameter from the `search_by_position` endpoint